server/sonar-server/src/main/java/org/sonar/server/issue/notification/NewIssuesEmailTemplate.java

```diff
@@ -42,13 +44,24 @@
   public static final String FIELD_PROJECT_NAME = "projectName";
   public static final String FIELD_PROJECT_KEY = "projectKey";
   public static final String FIELD_PROJECT_DATE = "projectDate";
+  public static final String FIELD_PROJECT_UUID = "projectUuid";
 
   private final EmailSettings settings;
   private final I18n i18n;
+  private final UserIndex userIndex;
 
-  public NewIssuesEmailTemplate(EmailSettings settings, I18n i18n) {
+  public NewIssuesEmailTemplate(EmailSettings settings, I18n i18n, UserIndex userIndex) {
```

Comment:
Do you confirm that there are at most 5 calls to Elasticsearch to get user names ? Number of requests do not relate to number of issues ?